brackend/tasks/tasks.py

```diff
@@ -48,19 +50,45 @@ def save_new_user(username, firebase_id):
 
 
 def get_user_ids():
+    """
+    returns a list of all users in the database
+    """
     engine = EngineGetter.get_or_create_engine()
     with Session(engine) as session:
         users = session.query(User).all()
         return [u.id for u in users]
 
 
 def get_tournament_ids():
+    """
+    returns a list of all tournaments in the database
+    """
     engine = EngineGetter.get_or_create_engine()
     with Session(engine) as session:
         users = session.query(Tournament).all()
         return [u.id for u in users]
 
 
+def delete_tournament(tid, uid):
+    """
+    delete tournament from the database
+    """
+    engine = EngineGetter.get_or_create_engine()
+    with Session(engine) as session:
+        tournament = (
+            session.query(Tournament)
+            .join(Tournament.user_tournaments)
+            .join(UserTournament.user)
+            .where(Tournament.id == tid)
+            .where(User.firebase_id == uid)
+            .where(UserRole.organizer == UserTournament.role)
+        )[0]
+        assert tournament is not None
```

Comment:
Looks good, just one small thing here: You should use `.one_or_none()` instead of accessing the first element like `[0]` (which would error out before even getting to the assert on 86)